travel-event-host-app/src/components/header/Header.tsx

```diff
@@ -1,15 +1,22 @@
 'use client';
+import theme from '@/app/theme';
+import Language from '@/lib/language';
+import { Box, FormControl, MenuItem, Select, SelectChangeEvent } from '@mui/material';
+import CircularProgress, { circularProgressClasses } from '@mui/material/CircularProgress';
+import Link from 'next/link';
 import { useState } from 'react';
+import HeaderBarAvatar from '../avatar/header-bar-avatar/HeaderBarAvatar';
 import styles from './styles.module.css';
-import Link from 'next/link';
-import Avatar from '@mui/material/Avatar';
-import CircularProgress, { circularProgressClasses } from '@mui/material/CircularProgress';
-import Language from '@/lib/language';
 export default function Header() {
   const [lang, setLang] = useState<Language>(Language.En);
   const [status, setStatus] = useState('unauthenticated');
   const [navMenuIsOpen, setnavMenuIsOpen] = useState(false);
   const userName = 'Angelo';
+
+  const handleLanguageOptionChanged = (event: SelectChangeEvent) => {
```

Comment:
why is better have a handle event if just set the state?

Reply:
Yes, no need. I'll update